packages/services/src/serverconnection.ts

```diff
@@ -334,7 +334,14 @@ namespace Private {
    */
   function getCookie(name: string): string | undefined {
     // From http://www.tornadoweb.org/en/stable/guide/security.html
-    const matches = document.cookie.match('\\b' + name + '=([^;]*)\\b');
+    let cookie = '';
+    try {
+      cookie = document.cookie;
```

Comment:
it would also be a reasonable spec to let this raise here and catch where getCookie is called. It is called in exactly one place, to get the xsrf token.